src/components/textbox/textbox.spec.js

```diff
@@ -36,6 +38,37 @@ describe("Textbox", () => {
     expect(wrapper).toMatchSnapshot();
   });
 
+  it("renders a counter", () => {
+    const wrapper = mount(<Textbox value="test string" characterLimit="100" />);
```

Comment:
I wonder, maybe it would be worth to do
```js
const characterLimit = "100"
const value = "test string"
```

```js
expect(wrapper.find(CharacterCount).text()).toBe(value.length '/' characterLimit);
``` 
or at least
```js
expect(wrapper.find(CharacterCount).text()).toBe(`${value.length}/100`);
```

So we can see that it should be `11` or other number (if different value used) without reading the value and counting characters to be sure we didn't make any mistake

This is just an idea, it's up to u